### DISCUSSION TITLE
Guys, I want to expand the click area, how to achieve?

### DISCUSSION THREAD

**Discussion by yigeqiji** [CaiJimmy/hugo-theme-stack#668] [Q&A]
![image](https://user-images.githubusercontent.com/59365443/187139652-2b7be4ba-48c6-4389-bd43-e16aaf4cce92.png)

**Comment by rmdhnreza** [comment]
I think just move `a` Tag from `article-title` in `/layouts/partials/article/components/details.html`

https://github.com/CaiJimmy/hugo-theme-stack/blob/72c5f4ea2488480bb71734eb0b0a9719def38ee7/layouts/partials/article/components/details.html#L13-L17

1. Cut `<a href="{{ .RelPermalink }}">` tag from `Line 14` to `Line 11`
2. Cut `</a>` closing tag from `Line 16` to `Line 57`

![a](https://user-images.githubusercontent.com/23427492/187170006-74f2a758-9984-470e-a3ae-f335905dd048.png)
